qcengine/config.py

```diff
@@ -222,12 +222,13 @@ def get_config(hostname=None, local_options=None):
 
 def get_provenance():
     from qcengine import __version__
-    ret = {
-        "cpu": GLOBALS["cpu"],
-        "hostname": GLOBALS["hostname"],
-        "username": GLOBALS["username"],
-        "qcengine_version": __version__
-    }
+    ret = Provenance(
+        cpu=GLOBALS["cpu"],
+        hostname=GLOBALS["hostname"],
+        username=GLOBALS["username"],
+        version=__version__,
```

Comment:
We cannot use `version` here as it will overwrite the programs version. Please revert and remove the creator as well below.

Reply:
I think I fixed this, the behavior should reflect what is in live right now. I think this Provenance's `Creator` still overrides the output program creator, but it does that now. Unsure if thats a bug or on purpose.

On closer look, maybe it does still overwrite the version. Should we instead be updating the Provenance as `QCEngineProvenance.update(SourceProgramProvenance)` instead of the other way around like we do now?

Reply:
We previously did not set the creator field so this wasn't an issue. This is mostly extra interesting information that had nowhere else to go and should not overlap with the source program provenance.